lib/vanagon/platform/dsl.rb

```diff
@@ -161,6 +161,19 @@ def servicetype(type)
         @platform.servicetype = type
       end
 
+      # Set the list of possible host to perform a build on (when not using
+      # pooler or CLI flags)
+      #
+      # @param type [Array] the names of the hosts (must be resolvable)
+      def build_hosts(*args)
+        raise ArgumentError if args.empty?
```

Comment:
Also having some kind of text in the ArgumentError would be good, so the error is meaningful.

The yard doc should also mention the `@raise`.